app/views/properties/show.html.erb

```diff
@@ -1,45 +1,54 @@
 
 <div id="content">
-  <div class="phase-banner">
-    <p>
-      <strong class="phase-tag">BETA</strong>
-      <span>This is a design and user research prototype. <a href="https://www.gov.uk/service-manual/phases">Find out more about service design phases</a>.</span>
-    </p>
+  <div class="page-section top-section">
+    <div class="phase-banner">
+      <p>
+        <strong class="phase-tag">BETA</strong>
+        <span>This is a design and user research prototype. <a href="https://www.gov.uk/service-manual/phases">Find out more about service design phases</a>.</span>
+      </p>
 
-  </div>
-  <div class="header-context" id="global-breadcrumb">
-    <nav role="navigation">
-      <ol class="group">
-        <li><a href="#"></a></li>
-        <li><a href="#"></a></li>
-      </ol>
-    </nav>
-  </div>
+    </div>
+    <div class="header-context" id="global-breadcrumb">
+      <nav role="navigation">
+        <ol class="group">
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+        </ol>
+      </nav>
+    </div>
 
 
-  <div class="inner-block">
-    <div class="grid-row">
-      <div class="column-half property_details">
-        <h1 class="property-details-title-prefix">Property details</h1>
-        <% @property[:address].each do | address_line| %>
-          <h2 class="address"><%= address_line %></h2>
-        <% end %>
+    <div class="inner-block">
+      <div class="grid-row">
+        <div class="column-half property_details">
+          <h1 class="property-details-title-prefix">Property details</h1>
+            <h2 class="address">@property[:address]</h2>
```

Comment:
We should not be removing the loop that was in place for `@property[:address]` :(